comint-mime.el

```diff
@@ -137,13 +142,13 @@ from `comint-mode', or interactively after starting the comint."
 (defun comint-mime-render-svg (header data)
   "Render SVG from HEADER and DATA provided by `comint-mime-osc-handler'."
   (let ((start (point)))
-    (insert-image (svg-image data))
+    (insert-image (apply (apply-partially #'svg-image data) comint-mime-default-image-params))
```

Comment:
Why not:
```suggestion
    (insert-image (apply #'svg-image data comint-mime-default-image-params))
```

Reply:
It looks like it works for both as well.